docs/components/checkbox-field/index.md

```diff
@@ -308,3 +312,22 @@ Target the error block via `data-error`.
     @isDisabled={{true}}
   />
 </div>
+
+### CheckboxField with isReadOnly
```

Comment:
I see the `index.md` is updated here but not for checkbox-group-field?

Reply:
Nice catch - got docs all updated at https://github.com/CrowdStrike/ember-toucan-core/pull/132/commits/00cf88e3db95c947bf634a20208ef63e63c9c9bb